src/Analyzer/Factory.php

```diff
@@ -6,49 +6,127 @@
 namespace PHPSA\Analyzer;
 
 use PHPSA\Analyzer;
+use PHPSA\Configuration;
+use Symfony\Component\Config\Definition\Builder\NodeDefinition;
 use Webiny\Component\EventManager\EventManager;
 use PHPSA\Analyzer\Pass as AnalyzerPass;
 
 class Factory
```

Comment:
Please protect whole static methods, I dont think that static methods is needed
Thanks